scripts/postprocessing.py

```diff
@@ -10,4 +10,46 @@ def hello_metriq():
   print(client.hello())
   # >>> {'status': 'API is working', 'message': 'This is the Metriq public REST API.'}
 
-hello_metriq()
\ No newline at end of file
+# hello_metriq()
+
+def process_results():
+  folder_path = os.path.abspath(os.path.join( os.path.dirname( __file__ ),"..", "results"))
+  filenames = os.listdir(folder_path)
+  for filename in filenames:
+    with open(os.path.join(folder_path, filename), "r") as file:
+      content = file.read()
+      print(content)
+    print("-" * 25)
+
+  # TODO:
+  # 2. compute ave and std for each file (one for each architecture)
+  # 3. create result item and add to client
+  # 4. create metriq submission item
+  # 5. submit item though metriq API
+
+  # client = MetriqClient(token=METRIQ_TOKEN)
+  # methods = client.method_names_get()
+  # print(*methods, sep = "\n")
+
+  # print(client.task_names_get()) #400: Request validation error
+  # print(client.task_get("26")) #ex1_226.qasm (Aspen) #400: Request validation error
+  # print(client.task_get("27")) #ex1_226.qasm (Rochester) #400: Request validation error
```

Comment:
@WrathfulSpatula @vprusso 
Is this the correct way to call task 25's children tasks: [26](https://metriq.info/Task/26) and [27](https://metriq.info/Task/27)? I am getting a 400 error.
I am trying to populate those tasks using the API, and was unsure what values are expected for those under "task" param in the result item example below.

Reply:
Hi @karlaspuldaro 

Thanks for the update and information. Indeed, you are correct that the task endpoints were misbehaving. 

I've gone ahead and put in a hotfix for this issue here (https://github.com/unitaryfund/metriq-client/pull/78).

If you are able to update your metriq-client package with this fix that's since gone into `main`, you should be all set. Of course, let me know if that's not that case!

Reply:
Hi @vprusso 
Thanks for looking at it, although the task functions still give me an error when trying to return something typed Task. The traceback origin seems to be coming from 
```
File "site-packages/tea_client/handler.py", line 33, in wrapper
    raise ValidationError(error=e)
tea_client.errors.ValidationError: ValidationError(400: Request validation error.)
```
For now I am able call `client .http.get(f"/task/names/")` and `client.http.get(f"/task/{task_id}/")` and inspect `response["data"]`.

Reply:
Hi @karlaspuldaro, 

Thanks for the response. When you say:

>  ...the task functions still give me an error when trying to return something typed Task. 

I'm not quite sure I follow. Is there a specific task route that is causing an error? They appear to be working for me, but if you can provide me an explicit example of something that isn't working, I can take a look at that.

Reply:
@vprusso I still have the same validation errors when calling 
```
client.task_names_get() # ValidationError(400: Request validation error.)
client.task_get("26") # ValidationError(400: Request validation error.)
client.task_get("27") # ValidationError(400: Request validation error.)
```
My workaround is to call those task endpoints using `client.http.get(f"/task/names/")` and `client.http.get(f"/task/{task_id}/")` directly.

I am not sure I have the latest version, and this could be why these are not working for me but they are for you.
Even though I am installing metriq-client from github master branch, when inspecting the version with
```
from metriq.version import __version__
print("metriq-client version: ", __version__)
```
I get 0.1.2, and not 0.2.0 as in latest tag

Reply:
Hi @karlaspuldaro. Hmm, yes it might be that you need to run

```
pip install --upgrade metriq-client
```

in your project to incorporate the changes from the aforementioned hotfix branch that I put in from [metriq-client-#78](https://github.com/unitaryfund/metriq-client/pull/78). 

If you update the `metriq-client` project and then run something minimal like:

```py
from metriq import MetriqClient

client = MetriqClient(token="<METRIQ-TOKEN-API>")
result = client.task_names_get()
print(result)
```

the 400 validation errors should not be thrown. Of course, let me know if that's not the case.

Reply:
I haven't notice the default branch in metriq-client github is `develop`, and since was installing it from `master` I didn't get your fix.
Running `pip install --upgrade https://github.com/unitaryfund/metriq-client/tarball/development` worked for me!

Reply:
Excellent, glad to know it's working now! Let us know if we can provide any additional support and assistance, @karlaspuldaro